Cargo.toml

```diff
@@ -38,7 +38,7 @@ array-init = { version = "2.0", default-features = false }
 blake2 = { version = "0.10" }
 cargo_metadata = { version = "0.19.0" }
 cfg-if = { version = "1.0" }
-contract-build = { git = "https://github.com/use-ink/cargo-contract", branch = "master" }
+contract-build = { git = "https://github.com/use-ink/cargo-contract", branch = "alex/update-dependencies" }
```

Comment:
nit: Does something (new) break with this pointing to master? 
https://github.com/use-ink/cargo-contract/pull/2009 doesn't appear to change `contract-build` in a breaking way.